Sources/OpenAPIRuntime/Base/OpenAPIValue.swift

```diff
@@ -143,6 +143,18 @@ public struct OpenAPIValueContainer: Codable, Equatable, Hashable, Sendable {
             try container.encode(value.map(OpenAPIValueContainer.init(validatedValue:)))
         case let value as [String: OpenAPIValueContainer?]:
             try container.encode(value.mapValues(OpenAPIValueContainer.init(validatedValue:)))
+        case let value as [Sendable?]:
+            try container.encode(
+                value.map {
+                    try OpenAPIValueContainer.init(unvalidatedValue: $0)
+                }
+            )
+        case let value as [String: Sendable?]:
+            try container.encode(
+                value.mapValues {
+                    try OpenAPIValueContainer.init(unvalidatedValue: $0)
+                }
+            )
```

Comment:
Couple of suggestions here:

- We'll be landing #22 soon which means that you'll need to use existential any, i.e. `any Sendable` in place of `Sendable`, so we may as well get it right here so it doesn't need updating again.
- The rest of the switch statement follows a different style of passing `init(unvalidatedValue:)` to `map` and `mapValues` directly, so could we keep it consistent with these new switch cases?

```suggestion
        case let value as [(any Sendable)?]:
            try container.encode(value.map(OpenAPIValueContainer.init(unvalidatedValue:)))
        case let value as [String: (any Sendable)?]:
            try container.encode(value.mapValues(OpenAPIValueContainer.init(unvalidatedValue:)))
```